views/feed.handlebars

```diff
@@ -28,11 +28,9 @@
     <ul>
```

Comment:
Might want to change this to a section of divs rather than list and make the whole div the link to the post